packages/project-access/src/index.ts

```diff
@@ -1,6 +1,9 @@
 export { FileName } from './constants';
 export { getFilePaths } from './file';
 export {
+    CAP_APPS_FOLDER,
```

Comment:
Could we adopt the standard casing here?

Reply:
Hmm since lint hasn't complained I guess its ok...would be nice to align with the rest of open-ux-tools for exported `consts`

Reply:
Yeah, I wasn't sure what the alignment would be though. There are quite a few examples of uppercase with underscores across the modules?